react/Payment.tsx

```diff
@@ -21,22 +22,41 @@ const Payment: React.FC = () => {
   } = useOrderPayment()
   const history = Router.useHistory()
 
-  const onCardFormCompleted = (cardDigits: string) => {
-    setCardLastDigits(cardDigits)
+  const goToCardForm = () => {
+    setStage(PaymentStage.CARD_FORM)
+  }
+
+  const goToInstallments = () => {
     setStage(PaymentStage.INSTALLMENTS)
   }
 
-  const onInstallmentSelected = async (installment: number) => {
+  const goToPaymentList = () => {
+    setStage(PaymentStage.PAYMENT_LIST)
+  }
+
+  const handleNewCreditCard = () => {
+    setCardType('new')
+    goToCardForm()
+  }
+
+  const handleSavedCreditCard = (payment: AvailableAccount) => {
+    setPaymentField({
```

Comment:
```suggestion
  const handleSavedCreditCard = async (payment: AvailableAccount) => {
    await setPaymentField({
```

Reply:
Thanks. I keep forgetting this `setPaymentField` is not a state setter...